homeassistant/components/meteo_france/config_flow.py

```diff
@@ -52,15 +53,18 @@ async def async_step_user(self, user_input=None):
         latitude = user_input.get(CONF_LATITUDE)
         longitude = user_input.get(CONF_LONGITUDE)
 
-        client = MeteoFranceClient()
-
         try:
+            client = MeteoFranceClient()
+
             if not latitude:
                 places = await self.hass.async_add_executor_job(
                     client.search_places, city
                 )
                 _LOGGER.debug("places search result: %s", places)
                 return await self.async_step_cities(places=places)
+        except AbortFlow as exp:
+            _LOGGER.error(exp)
+            return self.async_abort(reason="already_configured")
```

Comment:
this should be removed, the check is done in `self._abort_if_unique_id_configured()` 😉

Reply:
We should discuss this. Because if I remove this code, the test is KO. The exception reason visible by the test is "unknown"

Reply:
Proposal here : #20

Reply:
OK more simple